package.json

```diff
@@ -66,7 +66,7 @@
     "semantic-release": "^15.13.3"
   },
   "peerDependencies": {
-    "semantic-release": ">=11.0.0 <16.0.0"
+    "semantic-release": ">=11.0.0 <17.0.0"
```

Comment:
Probably less than 18 here @juliuscc Or `<=`. The latest Semantic Release is 17.0.4

Reply:
Ah that is right!

Reply:
That is fixed now

Reply:
Will you update the beta release as well? @juliuscc

Reply:
Yes I will do that so you can try before I merge. 2 min